tests/client/amo/components/TestAddonDetail.js

```diff
@@ -313,6 +317,34 @@ describe('AddonDetail', () => {
       rootNode.querySelector('.InstallButton-switch input').disabled);
   });
 
+  it('disables install switch and uses button for OpenSearch plugins', () => {
+    const AddSearchProviderSpy = sinon.stub(window.external,
+      'AddSearchProvider');
+    const rootNode = renderAsDOMNode({
+      addon: { ...fakeAddon, type: ADDON_TYPE_OPENSEARCH },
+    });
+    const installButton = rootNode.querySelector('.InstallButton-button');
+    assert.equal(installButton.textContent, 'Add to Firefox');
```

Comment:
This logic lives in `<InstallButton>` not `<AddonDetail>` so I think it would make more sense to move these tests to `TestInstallButton`